do_freebsd.sh

```diff
@@ -61,13 +58,12 @@ mkdir ${BUILD_DIR}
 	2>&1 | tee cmake.log
 
 echo -n "start building: "; date
-printenv
 
 cd ${BUILD_DIR}
-  gmake -j$CPUS V=1 VERBOSE=1 
+  gmake -j$NPROC V=1 VERBOSE=1 
```

Comment:
i am not sure how is this related to "Use the GNU binutils linker".

Reply:
@tchaikov 
That snuck in under the radar, since it is a typo/difference between what is actually set in the top of the file....
But I'll first try to see if your PR works for FreeBSD.

Major problem is that the FreeBSD ports infrastructure also sets quite a few CMake variables on calling `cmake`, so I really need to work around it, and doing it in the Ceph-config will be better resistant against future changes on the FreeSD side.

Reply:
i think we discussed this before. my understanding is that, ceph tries to offer the machinery to work on different arch/distro/OS, and to avoid hardwiring any of them unless necessary. i think what you need is just make sure FreeBSD's ceph port use the machinery.

Reply:
@tchaikov 
Since  #30781 does not fix the FreeBSD problem.
Can I at least get this one in?

I've removed changes to do_freebsd.sh, so this is now a clean patch

Reply:
could you change the title to something like

> cmake: use GNU linker on FreeBSD

?

Reply:
@tchaikov 
Ping?